LearningJourney/Modules/Core/UI/UI/Resources/Assets.xcassets/Colors/CardBackground.colorset/Contents.json

```diff
@@ -5,9 +5,9 @@
         "color-space" : "srgb",
         "components" : {
           "alpha" : "1.000",
-          "blue" : "0xF7",
-          "green" : "0xF1",
-          "red" : "0xF1"
+          "blue" : "0.969",
```

Comment:
Did you mean to change those values?

Reply:
Actually I didn't, must have been some kind of mistake

Reply:
maybe thats whats breaking ci